pkg/util/errors.go

```diff
@@ -92,20 +99,25 @@ var (
 	ErrAlreadyExistsInStorage = errors.New("unique constraint violation")
 )
 
+// ErrBadRequestStorage represents a storage error that should be translated to http.StatusBadRequest
 type ErrBadRequestStorage error
 
 // HandleStorageError handles storage errors by converting them to relevant HTTPErrors
-func HandleStorageError(err error, entityName string) error {
+func HandleStorageError(err error, entityNames ...string) error {
```

Comment:
always index 0 is used. `entityName` might be better and godoc should be adjusted to say that if entity name is provided the first one will be used

Reply:
i Think its always used with arg now so i will remove the varargs